src/file.ts

```diff
@@ -2295,11 +2295,14 @@ class File extends ServiceObject<File> {
    *     'v2' or 'v4.
    * @param {string} [config.cname] The cname for this bucket, i.e.,
    *     "https://cdn.example.com".
-   * @param {string} [config.contentMd5] The MD5 digest value in base64. If you
-   *     provide this, the client must provide this HTTP header with this same
-   *     value in its request.
-   * @param {string} [config.contentType] If you provide this value, the client
-   *     must provide this HTTP header set to the same value.
+   * @param {string} [config.contentMd5] The MD5 digest value in base64. Just like
+   *     if you provide this, the client must provide this HTTP header with this same
+   *     value in its request, so to if this parameter is not provided here,
+   *     the client must not provide any value for this HTTP header in its request.
+   * @param {string} [config.contentType] Just like if you provide this, the client
```

Comment:
Should this read:

> If you provide this, the client must provided the corresponding HTTP header in its request. If no header is provided, the client must similarly not provide a header in the request.

Reply:
I am trying to clarify the docs and in the same time to not exclude options.
The issue here is signature calculation. If these params are not provided, there values, for the sake of signature, default to `''`. Having this in mind, not providing this param in the method and providing same header with `undefined` or `''` value in the request will work, that's why I wrote 

> if this parameter is not provided here,
> the client must not provide any value for this HTTP header in its request

Hinting that providing an "empty" header is OK. (Since user may use third party modules which may work differently).

How does below sounds?

> If you provide this value, the client must provide a corresponding HTTP header set to the same value, if this value is not provided here, the client must similarly not provide this header in the request.